shell/platform/windows/angle_surface_manager.cc

```diff
@@ -169,14 +170,26 @@ void AngleSurfaceManager::CleanUp() {
   }
 }
 
-bool AngleSurfaceManager::CreateSurface(WindowsRenderTarget* render_target) {
+bool AngleSurfaceManager::CreateSurface(EGLint width, EGLint height) {
+  return CreateSurface(current_render_target_, width, height);
+}
+
+bool AngleSurfaceManager::CreateSurface(WindowsRenderTarget* render_target,
```

Comment:
What's the reason for this division of methods? This isn't static, and only ever called with `current_render_target_` unless I missed a call, so the implementation could just be folded into the method above and use `current_render_target_` directly.

Reply:
Fixed in next commit

Reply:
This is still taking the render target as an argument even thought the target is an ivar now though.